packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -256,12 +265,26 @@ watch(search, (newVal) => {
 
 watch(areQueriesLoading, (newVal) => (showLoadingBar.value = newVal))
 
-const twoYearsFromNow = new Date()
-twoYearsFromNow.setFullYear(twoYearsFromNow.getFullYear() + 2)
+function getFutureDateByDays(daysToAdd: number) {
+  const futureDate = new Date()
+  futureDate.setDate(futureDate.getDate() + daysToAdd)
```

Comment:
What if getDate() returns 31? If you add to it, is it gonna roll over back to 1?

Generally I advise using dayjs() for any date manipulation, cause it takes care of all kinds of details like this. The base JS Date() object and its related functions are generally way too low level to easily work with.